bundles/SyliusResourceBundle/configuration.rst

```diff
@@ -18,7 +18,6 @@ In your ``app/config.yml`` (or in an imported configuration file), you need to d
         resources:
             my_app.entity_key:
                 driver: doctrine/orm
-                manager: default
```

Comment:
Actually I tried renaming it to object_manager with no success. 

[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]             Unrecognized option "object_manager" under "sylius_resource.resources.my_app.entity_key"